### ISSUE TITLE
Is scrolling the page while operating or receiving focus a context change? (3.2.1, 3.2.2, 3.2.5)

### ISSUE THREAD

**Issue by JAWS-test** [w3c/wcag#2542]
The definition of change of context does not mention unexpected scrolling. Should that not be included or is that meant indirectly?

Examples:
* A link gets focus and is scrolled outside the visible area.
* When keyboard navigating through a radio button group, it is scrolled into the invisible area.

Are these not context changes? If no: Do they violate a current criterion (e.g. 2.4.7). But then what about cases where scrolling occurs, but not outside the visible area: only slightly annoying with keyboard operation, but very problematic when I work with a screen magnifier. This would not be a violation of 2.4.7, but also an accessibility problem...

I'm not talking about expected scrolling: when something that is currently not visible and gets focus is displayed by scrolling in the visible area.

**Comment by GreggVan** [comment]
Here is feedback on this from when we created this language (from my memory of it)

There was a lot of discussion about this and - as you can guess - it is easy to describe the extremes and hard to describe the middle ground

The underlying thought was —  "if you made a change in the page that a sighted person would know about - but a blind or low vision (zoomed in) user would not see or notice — then you should indicate that such a change occurred. 

Changes of Context
Jumping to a new page
Rewriting the same page with all new content
Changing the text on the page above the point where the person is

Not changes of context 
Expanding text just below the point of interaction
Changing the status indication of the control you are on

Harder to judge
Rewriting bits of the page
Changing text on the page below the point - where the person may have already read that text and would not know that it has now changed.

**Comment by fstrr** [comment]
This issue looks like it can be closed. If it needs to be re-opened, please do so and convert it to a discussion.
